sustainability-best-practices/frontend/image-optimisation.md

```diff
@@ -1,9 +1,28 @@
 # Image optimization
 
-Images typically make up [42% of the Largest Contentful Paint](https://paulcalvano.com/2021-06-07-lcp-httparchive/) element for websites. It's therefore vital that we optimize these before uploading to our website. This results in a smaller footprint for a user to download, but also will use less computing to upload.&#x20;
+Images typically make up [42% of the Largest Contentful Paint](https://paulcalvano.com/2021-06-07-lcp-httparchive/) element for websites, responsible for the largest over-the-wire transfer rates. We can reduce their size and deliver them with optimization in mind.
 
-We can make use of better image formats. It could be AV1 Image File Format (AVIF) and WebP to see 50% and 26% smaller file sizes than JPEGs respectively. AVIF is less well supported than Webp so it’s important to use a **picture tag** to ensure compatibility. You can read more about this in the [Using Modern Image Formats: AVIF And WebP](https://www.smashingmagazine.com/2021/09/modern-image-formats-avif-webp/) article.
+## Optimize before upload
 
-The other benefit of using a picture tag is that we can denote which images to use based on screen size. This is helping us to provide the most appropriate image to the user.
+It's important that we optimize these **before uploading** to our website. This results in a smaller footprint for a user to download, but also will use less computing to upload. Using tools such as [Squoosh](https://squoosh.app/) or resizing to an appropriate maximum expected size that the image will be displayed on the website will help reduce the overall emissions during the website lifetime.
```

Comment:
to keep our styleguide rules to having a sentenced contain less than 25 words to make the docs more readable: https://docs.umbraco.com/welcome/contribute/style-guide#long-sentences
```suggestion
It's important that we optimize these **before uploading** them to our website. This results in a smaller footprint for a user to download, but will also use less computing to upload. Tools such as [Squoosh](https://squoosh.app/) or resizing to an appropriate maximum expected size can be used for optimizing. Then the images displayed on the website will help reduce the overall emissions during the website's lifetime.
```